format_indirect.md

```diff
@@ -0,0 +1,62 @@
+# Indirect Format
+
+The Indirect Format allows for a dependency to be referenced in a smaller format that takes up less space and bandwidth. This is designed primarily for the `src` field in a dependency.
+
+An indirect project always starts with the `"indirect": true` value in order to clearly indicate that it's indirect. This allows launchers and clients to understand and treat this differently.
```

Comment:
What different treatment is needed?

Reply:
Launchers have to explicitly look for the `full` field in versions to find the full metadata. But the special indication may not be needed - launchers can just check for the existence of the `full` field, and assume it's indirect.